api/util/errorLog-function.js

```diff
@@ -0,0 +1,31 @@
+const ErrorLog = require('../models/ErrorLog');
+
+async function addErrorLog(req){
+  let errorSaved = true;
+  const newError = new ErrorLog({
+    userEmail: req.body.userEmail,
+    errorTime: req.body.errorTime,
+    apiEndpoint: req.body.apiEndpoint,
+    errorDescription: req.body.errorDescription
+  });
+  await newError.save()
+    .catch(_ => {
+      errorSaved = false;
+    });
+  return errorSaved;
+}
+/**
+ * Add a new error.
```

Comment:
youre supposed to paste the documentation **above the function**. Look inside of any file in [`src/APIFunctions`](https://github.com/SCE-Development/Core-v4/tree/dev/src/APIFunctions)